dietpi.txt

```diff
@@ -161,6 +161,13 @@ CONFIG_CPU_MIN_FREQ=Disabled
 # Disable Intel-based turbo/boost stepping. This flag should not be required, setting <100% MAX frequency should disable Turbo on Intel CPUs.
 CONFIG_CPU_DISABLE_TURBO=0
 
+#GPU Driver | Will also be applied during 1st run if set to a value other than 'None'
+# - None = No GPU
+# - Intel
+# - Nvidia
+# - AMD
+CONFIG_GPU_DRIVER='None'
```

Comment:
We should make clear here that it is relevant only for x86_64 PC systems when using gaming or video software (has not really an effect on common desktop usage, right?).

For simplicity, shall we just expect lower-case characters everywhere? In the `dietpi-config` menu where it looks nicer, we can  `${G_WHIP_RETURNED_VALUE,,}`.

Reply:
> We should make clear here that it is relevant only for x86_64 PC systems when using gaming or video software

Also Chromium (Video decode and surface rendering) and any software that will use HW acelleration on the driver stack.

I would personally leave it as simply GPU driver. If needed,  add "Adds support for HW accell, for applications that support OpenGL/GLES/Vulkan and other API's etc".

> (has not really an effect on common desktop usage, right?).

Should take load of CPU (eg: moving windows etc). X11 will render them in OpenGL on GPU HW I believe?

Reply:
Could mention x86 only. But left it out as you mentioned ARM external support. Could be done in future?

Reply:
> Also Chromium (Video decode and surface rendering)

Jep, everything that does video playback I mean. OpenGL and GLES are installed with X11 in any case (hard package dependency), but indeed Chromium generally supports VA-API as well, AFAIR and at least for Nvidia and AMD the firmware and DDX will have an effect on desktop rendering. Ah and `nvidia-driver` extends OpenGL as well. Okay indeed just makes sense in general if any GUI application is used 👍.

Then I suggest:
```
# - x86_64 PC only!
# - Adds support for GUI/video hardware acceleration, OpenGL/GLES, Vulkan and VA-API
```